lib/glific/flows.ex

```diff
@@ -330,14 +331,13 @@ defmodule Glific.Flows do
     Flow.changeset(flow, attrs)
   end
 
-  defp get_user do
-    user = Repo.get_current_user()
-
-    {email, name} =
-      if user,
-        do: {"#{user.phone}@glific.org", user.name},
-        else: {"unknown@glific.org", "Unknown Glific User"}
+  defp get_user(nil) do
+    %{email: "unknown@glific.org", name: "Unknown Glific User"}
+  end
 
+  defp get_user(user_id) do
+    user = Repo.fetch(User, user_id) |> elem(1)
```

Comment:
Rather than fetching for each user, can we join the users table in the previous query. @AmishaBisht 
@AkhileshNegi thoughts?

Reply:
@mdshamoon not sure what u meant there